src/main/resources/swagger/api-docs.yaml

```diff
@@ -241,6 +241,240 @@ paths:
+      description: create new billing project in rawls
+      operationId: createBillingProjectFullV2
+      requestBody:
+        description: create project request
+        content:
+          'application/json':
+            schema:
+              $ref: '#/components/schemas/CreateRawlsBillingProjectFullRequest'
+        required: true
+      responses:
+        201:
+          description: Successfully Created Billing Project in Rawls
+        400:
+          description: both you and firecloud billing user must be a user of the billing account
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/ErrorReport'
+        409:
+          description: project already exists in rawls
+          content: {}
+        500:
+          description: Rawls Internal Error
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/ErrorReport'
+      security:
+        - authorization:
+            - openid
+            - email
+            - profile
+            - https://www.googleapis.com/auth/cloud-billing
+      x-codegen-request-body-name: createProjectRequest
+  /api/billing/v2/{projectId}:
+    get:
+      tags:
+        - billing_v2
+      summary: get billing project
+      description: get billing project
+      operationId: getBillingProject
+      parameters:
+        - name: projectId
+          in: path
+          description: Id of the billing project
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: OK
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/RawlsBillingProjectResponse'
+        404:
+          description: Project Not Found
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/ErrorReport'
+        500:
+          description: Internal Server Error
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/ErrorReport'
+    delete:
+      tags:
+        - billing_v2
+      summary: delete billing project
+      description: delete billing project
+      operationId: deleteBillingProject
+      parameters:
+        - name: projectId
+          in: path
+          description: Id of the billing project
+          required: true
+          schema:
+            type: string
+      responses:
+        204:
+          description: OK
+        404:
+          description: Project Not Found
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/ErrorReport'
+        500:
+          description: Internal Server Error
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/ErrorReport'
```

Comment:
Was there an intentional decision to decide to leave out `/api/billing/v2/{projectId}/billing-account`?  What was the reasoning behind deciding which APIs were added to Orch and which were not?

Reply:
There wasn't an intentional decision, I just did all the ones I could find in the api-docs in Rawls. Let me double check.

Reply:
Ah! Its cause that endpoint doesn't exist in Rawls yet, only on our feature branch.

Reply:
Updated the ticket to possibly cherry-pick that endpoint onto develop to include a note to update Orch as well

https://broadworkbench.atlassian.net/browse/CA-1140